pkg/registry/core/service/storage/rest.go

```diff
@@ -162,8 +218,44 @@ func (rs *REST) Export(ctx context.Context, name string, opts metav1.ExportOptio
 	return rs.services.Export(ctx, name, opts)
 }
 
+func (rs *REST) isExternalIPAM(ctx context.Context, service *api.Service) (bool, error) {
+	netName := service.Labels[arktosextensionsv1.NetworkLabel]
+	if netName == "" {
+		// choose the default network name if not specified
+		netName = "default"
```

Comment:
kinda feeling "default" will be a const used a lot in future work. Give it a const definition?

Reply:
will have a const as suggested.